CUDADataFormats/BeamSpot/BuildFile.xml

```diff
@@ -1,8 +1,10 @@
-<use name="rootcore"/>
-<use name="CUDADataFormats/Common"/>
-<use name="DataFormats/Common"/>
-<use name="HeterogeneousCore/CUDAUtilities"/>
+<iftool name="cuda"/>
```

Comment:
@fwyzard , please remove the `/` from here

Reply:
done, thank you